### DISCUSSION TITLE
My complete Obsidian workflow template for life management

### DISCUSSION THREAD

**Discussion by mathisgauthey** [ObsidianPublisher/obsidian-github-publisher#189] [Show and tell]
## Introduction

![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/46576952/e7db89c6-e10e-41af-8a44-7627a8c3295e)

As I promised to quite a few people on different places, I’m releasing a full blown guide of my Obsidian workflow. It has many cool features that I’ve seen people struggling with online :

1. Tasks and project management with journaling, workout logging and media tracking. Using daily and periodic notes.
2. Read-it-later with automated conversion to markdown and import inside your vault. Meaning all your past knowledge is inside Obsidian, available with a simple search alongside your notes.
3. Fast and simple one click upload obsidian publish alternative.

But there’s plenty more. It took me some time here and there during the past year to come up with this setup. And don’t worry, it is **easy to use** despite its awfully long name. But I’ll describe it with as much depth as I can for people who wants to fine tune it.

Obviously, it is merely just a starting point, with Obsidian you can (and you always should) do something that works well for **you**. And that means creating your **own** setup. But that doesn’t forbid you from using this one as a starting-point.

Links : 
- [My Complete Obsidian Workflow to Manage My Life](https://mathisgauthey.github.io/my-complete-obsidian-workflow-to-manage-my-life/)
- [Why Using Obsidian for Life Management](https://mathisgauthey.github.io/why-using-obsidian-for-life-management/)
- [Obsidian Publish Alternative, How to One-click Upload Your Notes on Your Own Website](https://mathisgauthey.github.io/obsidian-publish-alternative-how-to-one-click-upload-your-notes-on-your-own-website/)
- [A template vault available on Github to try it for yourself](https://github.com/mathisgauthey/obsidian-workflow-template)

## Core Features

- 🎨 Beautiful design using [AnuPpuccin: Personal theme for Obsidian](https://github.com/AnubisNekhet/anuppuccin).
- 🗃️ [P.A.R.A](https://fortelabs.com/blog/para/)[^1] folder organisation based on file actionability.
- ✅ [GTD](https://gettingthingsdone.com/)[^2] task management on par with a [full Todoist setup](https://todoist.com/fr/productivity-methods/getting-things-done), but with a simple and natural way of **writing tasks alongside notes**. You also get to keep your project tasks, notes and files archives when you finish a project, allowing you to access these informations in the future.
- 📅 Daily notes for task management, periodic notes for projects management (week, month, quarter and year).
- 📓 Journal section for each daily and periodic notes to write about anything that happened during a certain period of time.
- 💪 Simple workout log for storing your workout sessions but with an infinite possible personalized exercise library.
- 🎮 Media backlog Kanban board for tracking books, shows, movies or video games backlog, consuming media at the right pace, and archiving what you do. All without proprietary online services that hold you data.
- 🔗 Read-it-later / bookmark manager with automated markdown conversion for offline storage inside your vault using [Omnivore app](https://omnivore.app/). That means easy search for knowledge inside your vault across sources and personal notes.
- 📝 [Zettelkasten note-taking](https://everlaab.com/methode-zettelkasten-comment-prendre-des-notes-utiles/) on a flat structure inside `3-Resources` folder.
	1. 📎 Import sources automatically using [obsidian-omnivore](https://github.com/omnivore-app/obsidian-omnivore) plugin for an automatic markdown conversion of a webpage or document online or manually from a file or using [markdownload](https://github.com/deathau/markdownload).
	2. Highlight and take reference notes using [Omnivore app](https://omnivore.app/) or directly inside your vault using a simple markdown file.
	3. Write literature notes to summarise the source informations.
	4. Write permanent notes from key concepts directly inside literature notes using wiki-links.
- 🌐 One click upload of file and attachments to a [Hugo](https://gohugo.io/) website hosted on [Github Pages](https://pages.github.com/) thanks to [obsidian-github-publisher](https://github.com/ObsidianPublisher/obsidian-github-publisher) plugin.

**Comment by mathisgauthey** [comment]
@Lisandra-dev I was just thinking, maybe you could steal my regex, or my whole settings, to make a standard Hugo importer just like the one you created for mkdocs ? ✌️ I don't know if you'd like a FR for that.

**Reply by Mara-Li** [reply]
I don't understand what do you mean :<

**Reply by mathisgauthey** [reply]
I meant using my [config file](https://github.com/mathisgauthey/obsidian-workflow-template/blob/main/.obsidian/plugins/obsidian-mkdocs-publisher/data.json) to create a preset in your plugin for Hugo website uploads :

![Screenshot_20230816-091621.png](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/46576952/67f1e9c5-253d-4623-b751-d5fce51f37f5)

**Reply by Mara-Li** [reply]
Oh yeah, you can use the export button and make a PR on the repo: https://github.com/ObsidianPublisher/plugin-presets

**Reply by mathisgauthey** [reply]
Alright, I'll do it next time I get on my laptop!

**Comment by mathisgauthey** [comment]
## Introduction

Hi everyone.

In 2023, I shared my [obsidian-workflow-template](https://github.com/mathisgauthey/obsidian-workflow-template) on Github.

The "documentation" was some mere blog posts on my website :

- [My Complete Obsidian Workflow to Manage My Life](https://mathisgauthey.github.io/my-complete-obsidian-workflow-to-manage-my-life/)
- [Obsidian Publish Alternative, How to One-Click Upload Your Notes on Your Own Website](https://mathisgauthey.github.io/obsidian-publish-alternative-how-to-one-click-upload-your-notes-on-your-own-website/)
- [Why Using Obsidian for Life Management](https://mathisgauthey.github.io/why-using-obsidian-for-life-management/)

## What's New

I just wanted to let you know that I just released an **updated version** of the workflow based on my experience in the past few months.

It has now it's own [documentation](https://mathisgauthey.github.io/obsidian-workflow-template-docs/).

You can see most of the changes in details in the [release page](https://github.com/mathisgauthey/obsidian-workflow-template/releases). I did not get everything in the text as there was a lot of small plugin config tweaks, but any major part should be here.

## Features

If you want to try it, here's the core features :

- 🎨 Beautiful design using [AnuPpuccin: Personal theme for Obsidian](https://github.com/AnubisNekhet/anuppuccin).
- 🗃️ [P.A.R.A](https://fortelabs.com/blog/para/)[^1] folder organisation based on file actionability.
- ✅ [GTD](https://gettingthingsdone.com/)[^2] task management on par with a [full Todoist setup](https://todoist.com/fr/productivity-methods/getting-things-done), but with a simple and natural way of **writing tasks alongside notes**. You also get to keep your project tasks, notes and files archives when you finish a project, allowing you to access these informations in the future.
- 📅 Daily notes for task management, periodic notes for projects management (week, month, quarter and year).
- 📓 Journal section for each daily and periodic notes to write about anything that happened during a certain period of time.
- 🎮 Media backlog Kanban board for tracking books, shows, movies or video games backlog, consuming media at the right pace, and archiving what you do. All without proprietary online services that hold you data.
- 🔗 Read-it-later / bookmark manager with automated markdown conversion for offline storage inside your vault using [Omnivore app](https://omnivore.app/). That means easy search for knowledge inside your vault across sources and personal notes.
- 📝 [Zettelkasten note-taking](https://everlaab.com/methode-zettelkasten-comment-prendre-des-notes-utiles/) on a flat structure inside `3-Resources` folder.
    1. 📎 Import sources automatically using [obsidian-omnivore](https://github.com/omnivore-app/obsidian-omnivore) plugin for an automatic markdown conversion of a webpage or document online or manually from a file or using [markdownload](https://github.com/deathau/markdownload).
    2. Highlight and take reference notes using [Omnivore app](https://omnivore.app/) or directly inside your vault using a simple markdown file.
    3. Write literature notes to summarise the source informations.
    4. Write permanent notes from key concepts directly inside literature notes using wiki-links.
- 🌐 One click upload of file and attachments to a [Hugo](https://gohugo.io/) website hosted on [Github Pages](https://pages.github.com/) thanks to [obsidian-github-publisher](https://github.com/ObsidianPublisher/obsidian-github-publisher) plugin.
